agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -97,13 +99,6 @@ private void createDiseaseAnnotationsFromOrthology() {
 			pairs.getRight().addAll(geneArrayListPair.getRight());
 		});
 		geneViaOrthologyMap = geneService.getOrthologousGeneDiseaseAnnotations(generatedImplicatedGeneMap);
-/*
-		Map<Gene, List<DiseaseAnnotation>> geneViaOrthologyMap = geneService.getOrthologousGeneDiseaseAnnotations(generatedImplicatedGeneMap);
-		geneViaOrthologyMap.forEach((gene, diseaseAnnotations) -> {
-			List<DiseaseAnnotation> das = geneViaOrthologyMap.computeIfAbsent(gene, gene1 -> new ArrayList<>());
-			das.addAll(diseaseAnnotations);
-		});
-*/
```

Comment:
Can we remove commented out code

Reply:
This is being removed!

Reply:
But its still here in the file?

Reply:
I am not sure what you mean...The commented out lines have been removed.